### ISSUE TITLE
Format audio cue doesn't always play

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode#196385]
Testing #196126

1. `"audioCues.format": "always"`
2. Enable format on save
3. Make some changes in a file that will mess up formatting
4. ctrl+s to save
5. File is formatted, but audio cue isn't played. I can hear the audio cue if I explicitly run the "Format Document" command.

**Comment by TylerLeonhardt** [comment]
Same when Format on Paste is enabled

**Comment by meganrogge** [comment]
And in these cases, there's formatting that happens to the document?

**Comment by TylerLeonhardt** [comment]
Yes. A very simple case:

```
const a= 4;
```

notice there's no space between `a` and `=`. When I paste, I see the code get formatted, but no sound.

**Comment by meganrogge** [comment]
Are you on Linux @TylerLeonhardt ? where you've reported audio cues do not work?

**Comment by TylerLeonhardt** [comment]
For me, yes, but @alexr00 is on Windows.

**Comment by meganrogge** [comment]
Let's track the linux case separately 👍 .

@alexr00 a gif would be helpful because I cannot reproduce. Also, if you could toggle keyboard shortcuts troubleshooting, that would help.

https://github.com/microsoft/vscode/assets/29464607/50c95c11-28f7-4f32-8413-119dbf352b4e

**Comment by alexr00** [comment]
Shared recording and keyboard shortcuts logs offline.

**Comment by meganrogge** [comment]
@alexr00 in your gif, there was never a dirty indicator on the file. do you have auto save enabled?

**Comment by meganrogge** [comment]
I still cannot repro for that specific case, but think I see the issue.
